sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -141,15 +153,24 @@ def test_Beam():
     q = x*(E*I*f - w0*SingularityFunction(e, a1, 4)/24 - w2*SingularityFunction(e, c1, 2)/2)/(E*I) + (w0*SingularityFunction(x, a1, 5)/120 + w2*SingularityFunction(x, c1, 3)/6)/(E*I) + (E*I*(-c*f + d) + c*w0*SingularityFunction(e, a1, 4)/24 + c*w2*SingularityFunction(e, c1, 2)/2 - w0*SingularityFunction(c, a1, 5)/120 - w2*SingularityFunction(c, c1, 3)/6)/(E*I)
     assert simplify(p - q) == 0
 
-    b3 = Beam(9, E, I)
+    b3 = Beam(9, E, I, 2)
     b3.apply_load(value=-2, start=2, order=2, end=3)
     b3.bc_slope.append((0, 2))
     C3 = symbols('C3')
     C4 = symbols('C4')
+
     p = b3.load
     q = -2*SingularityFunction(x, 2, 2) + 2*SingularityFunction(x, 3, 0) + 4*SingularityFunction(x, 3, 1) + 2*SingularityFunction(x, 3, 2)
     assert p == q
 
+    p = b3.shear_force()
+    q = -2*SingularityFunction(x, 2, 3)/3 + 2*SingularityFunction(x, 3, 1) + 2*SingularityFunction(x, 3, 2) + 2*SingularityFunction(x, 3, 3)/3
+    assert p == q
+
+    p = b3.shear_stress()
+    q = -1*SingularityFunction(x, 2, 3)/3 + 1*SingularityFunction(x, 3, 1) + 1*SingularityFunction(x, 3, 2) + 1*SingularityFunction(x, 3, 3)/3
```

Comment:
Please break these long lines with each line whitin 72 characters, using PEP8. Apply these to all long lines